composeApp/src/commonMain/kotlin/com/developersbreach/kotlindictionarymultiplatform/ui/components/UiStateHandler.kt

```diff
@@ -30,7 +30,7 @@ fun <T> UiStateHandler(
     Box(modifier = Modifier.fillMaxSize()) {
         when (uiState) {
             is UiState.Loading -> {
-                CircularProgressIndicator(
+                KdCircularProgressIndicator(
                     modifier = Modifier.align(Alignment.Center),
                     color = MaterialTheme.colorScheme.onBackground,
```

Comment:
It's good to have `MaterialTheme.colorScheme.onBackground` as default value so that every other call site doesn't have to provide it. I've applied that change.